### ISSUE TITLE
Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by Azioach** [DurtyFree/durty-cloth-tool#339]
I have the following error:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at QyidskhrMRmUy7PsINZ.Hs2W6Vh9c4x7l4hVTqn.D8Whxv6wG7()
   at QyidskhrMRmUy7PsINZ.Hs2W6Vh9c4x7l4hVTqn.mTiLSvuEp0(Object  , EventArgs  )
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

App Version: **3.0.0.8**
Project targets: **AltV**
DLC name: **nerp_civpack**
Collection name: ** // **
Project path: **C:\Users\zacka\OneDrive\Desktop\NERP\NERP_CLOTHING\Dirty_Projects\clothproject_nerp_civpack**
Output path: **C:\Users\zacka\OneDrive\Desktop\NERP\NERP_CLOTHING\Dirty_Projects**
Disable Extra Ymt Creation: **False**
Create Subfolder For Cloth Drawable Types: **False**
Replace settings enabled: **False // DLC: NoDlc**
Save File Version: **5**
```

Cloth files:
`[Please provide cloth files (ydd, ytd), cloth project and any other related files here + describe reproduction steps]`

**Comment by DurtyFree** [comment]
Fixed upcoming update

**Comment by DurtyFree** [comment]
Fixed in v3.1 https://github.com/DurtyFree/durty-cloth-tool/releases/tag/3.1.0.0
